tools/bpf2c/bpf_code_generator.cpp

```diff
@@ -16,6 +16,7 @@
 #include <iostream>
 #include <sstream>
 #include <vector>
+#include <format>
```

Comment:
move up into alphabetical order per our [development guide](https://github.com/microsoft/ebpf-for-windows/blob/main/docs/DevelopmentGuide.md#header-files)

Reply:
fixed